integrated_channels/sap_success_factors/transmitters/learner_data.py

```diff
@@ -28,5 +32,13 @@ class SuccessFactorsLearnerDataTransmitter(SuccessFactorsTransmitterBase):
     (taken from OCNWebServicesConfiguration and OCNWebServicesEnterpriseCustomerConfiguration) and the oauth token.
 
     Record information about the success/failure of posting the learner data to SuccessFactors in
-     CompletionStatusEventAudit
+    the LearnerDataTransmissionAudit object provided, and save it to the database.
+
     """
+    def transmit(self, learner_data):
+        """
+        Transmit the learner data payload to the SAP SuccessFactors channel.
+        """
+        # TODO ENT-221 will handle transmitting the learner data payload to the channel
+        # Question: do we only want to transmit if learner_data.course_completed == True?
```

Comment:
Now that I have a first pass at the transmit function (https://github.com/edx/edx-enterprise/pull/65), which is doing the most basic thing it can do, I think it might be easier to figure out how to handle when to send the payload.
SAP is expecting us to send this payload only once, when the user has completed a course. Using certificate generation as it stands in this PR, we are covering 3 out of 4 cases. The case we are not handling is audit/self-paced learners, but this might end up being a follow up item at this point.
It is unclear to me from reading through this PR if the LearnerDataTransmissionAudit object is getting persisted. I feel that the transmit function would ideally: 
1. Determine if an event was already sent for this learner/course, referencing the audit table.
2. Perform the call to SuccessFactors if this is a new event, else exit.
3. Save a new audit object reflecting the status of the call.
What do you think? If you agree I will add this into my implementation, but I want to make sure it makes sense given what you're doing.